src/main/java/redis/clients/jedis/BinaryClient.java

```diff
@@ -1244,6 +1256,43 @@ public void moduleUnload(final byte[] name) {
     return args;
   }
 
+  public void aclWhoAmi() { sendCommand(ACL, Keyword.WHOAMI.raw); }
+
+  public void aclGenPass() { sendCommand(ACL, Keyword.GENPASS.raw); }
+
+  public void aclList() { sendCommand(ACL, Keyword.LIST.raw); }
+
+  public void aclUsers() { sendCommand(ACL, Keyword.USERS.raw); }
+
+  public void aclCat() { sendCommand(ACL, Keyword.CAT.raw); }
+
+  public void aclCat(final byte[] category) {
+    sendCommand(ACL, Keyword.CAT.raw, category);
+  }
+
+  public void aclSetUser(final byte[] name) {
+    sendCommand(ACL, Keyword.SETUSER.raw, name);
+  }
+
+  public void aclGetUser(final byte[] name) {
+    sendCommand(ACL, Keyword.GETUSER.raw, name);
+  }
+
+  public void aclSetUser(final byte[] name, byte[][] parameters) {
+    final byte[][] cmdParams = new byte[2 + parameters.length][];
+    int index = 0;
+    cmdParams[index++] = Keyword.SETUSER.raw;
+    cmdParams[index++] = name;
+    for (final byte[] key : parameters) {
+      cmdParams[index++] = key;
+    }
```

Comment:
Please use joinParameters method.

Reply:
I have solved that in the commit 439d7aa692cd0ea4116d6307eefac2a76f5ca5f0